pages/changelog.mdx

```diff
@@ -21,11 +21,17 @@ Keep up to date on new product features and enhancements, API docs, tools change
 
 ## Completed
 
+### 2022-10-31
+
+**ADDED**
+
+- Added support for including [facilitator fee](/concepts/facilitator-fee) when creating transfers from Verified Customer's Bank into their Dwolla balance.
+
 ### 2022-10-27
 
 **ADDED**
 
-- Added the `correlationId` attribute to the [webhook payload](/concepts/webhook-events#webhook-payload) for transfer related webhooks if a value was specified on transfer creation.
+- Added the `correlationId` attribute to the [webhook payload](/concepts/webhook-events#webhook-payload) for transfer related webhooks if a value was specified on transfer creation. Previously, fees could only be applied to transactions between two parties.
```

Comment:
Should `Previously, fees could only be applied to transactions between two parties.` be included in the new changelog entry?

Reply:
lol I added it to the wrong spot! Thanks for catching that! xD

Reply:
Corrected [here](https://github.com/Dwolla/dev-portal/pull/439/commits/08d8a1f9dfa6c889f3e976111cac931891b39478).